src/bloqade/task/bloqade.py

```diff
@@ -89,6 +140,15 @@ def run(
 
 @BloqadeTask.set_serializer
 def _serialize(obj: BloqadeTask) -> Dict[str, Any]:
+    """
```

Comment:
I think these private methods are self-explainable, we don't have to add docstrings to them.